site/components/pages/Resources/Contact/styles.ts

```diff
@@ -1,12 +1,28 @@
 import { css } from "@emotion/css";
+import breakpoints from "@klimadao/lib/theme/breakpoints";
 
-export const section_textGroup = css`
+export const contactContainer = css`
   grid-column: main;
+
+  ${breakpoints.medium} {
+    display: grid;
+    grid-template-columns:
+      [contact_full-start] minmax(20rem, 1fr)
+      [contact_inner-start] minmax(0, 107.2rem)
```

Comment:
is this the same as the outer grid container?
If we do `grid-template-columns: inherit` we can keep the `full / main` convention everywhere